pkg/cmd/logs.go:583:22: undefined: getFromTargetInfo

**Describe the bug**

When i was checking concourse CI build log i found following error:
```
Cloning into 'gardenctl'...
# github.com/gardener/gardenctl/pkg/cmd
pkg/cmd/logs.go:583:22: undefined: getFromTargetInfo
```
https://concourse.ci.gardener.cloud/builds/2410#L5fb7906d:49

This issue is introduced in https://github.com/gardener/gardenctl/pull/431/commits/42c02e6866e7c1ec2ff27a21e164dbfb2e06939b#diff-59f23253994b1605ea49a120ea68de0c2ee3a4240c2825bb741f7b707695211fR582

the method `getFromTargetInfo(string)` has been changed to `func GetFromTargetInfo(targetReader TargetReader, key string)` in https://github.com/gardener/gardenctl/commit/16fc236f9ff027a465d4075eccb2c5c9dd9f9d21

@jguipi could you please look into this? if you need more information regarding above function please reach @tedteng 

Thanks.
-Neo




**To Reproduce**
check concourse ci log

**Expected behavior**
concourse ci build should success

**Screenshots**
![image](https://user-images.githubusercontent.com/42594392/99756498-44546580-2b28-11eb-94d5-12c97b416146.png)

**Gardenctl Version (please complete the following information):**
latest version in master branch
**Additional context**

--- a/pkg/cmd/logs.go
+++ b/pkg/cmd/logs.go
@@ -142,9 +142,9 @@ func runCommand(targetReader TargetReader, args []string) {
 		logsAddonManager(targetReader)
 	case "vpn-seed":
 		if len(args) == 2 {
-			logsVpnSeed(args[1])
+			logsVpnSeed(targetReader, args[1])
 		} else {
-			logsVpnSeed(emptyString)
+			logsVpnSeed(targetReader, emptyString)
 		}
 	case "vpn-shoot":
 		logsVpnShoot()
@@ -577,10 +577,10 @@ func saveLogsControllerManager(targetReader TargetReader) {
 }
 
 // logsVpnSeed prints the logfile of the vpn-seed container
-func logsVpnSeed(shootTechnicalID string) {
+func logsVpnSeed(targetReader TargetReader, shootTechnicalID string) {
 	fmt.Println("-----------------------Kube-Apiserver")
 	if shootTechnicalID == emptyString {
-		shootTechnicalID = getFromTargetInfo("shootTechnicalID")
+		shootTechnicalID = GetFromTargetInfo(targetReader, "shootTechnicalID")
 		logPodSeed("kube-apiserver", shootTechnicalID, "vpn-seed")
 	} else {
 		logPodSeed("kube-apiserver", shootTechnicalID, "vpn-seed")

